### ISSUE TITLE
incorrectly erroring on nullable date-time

### ISSUE THREAD

**Issue by SpencerLawrenceBrown** [cdimascio/express-openapi-validator#108]
Hi,

I think there is an issue with how nullable date-time fields are handled.

When a property is `type: string, format: date-time, nullable: true`, the null value is coerced into an empty string which fails the date-time validation.

I know this because if I remove the `format: date-time` in my example then 
```
{
     completedAt: null
}
```
becomes
```
{
    completedAt: ''
}
```
and passes validation.

Example that breaks:

`yaml`
```
WorkflowStep:
      properties:
        completedAt:
          description: If it hasnt been completed, value will be null.
          type: string
          nullable: true
          format: date-time
      required:
        - completedAt
```

`object`
```
{
    completedAt: null
 }
```

`error`
```
{ path: 'workflowStep.completedAt',
  message: 'should match format "date-time"',
  errorCode: 'format.openapi.validation' },
```

**Comment by SpencerLawrenceBrown** [comment]
I think I found the issue and I will open a PR

**Comment by cdimascio** [comment]
Thanks @SpencerLawrenceBrown. the fix is available in 2.14.2
